src/components/Home/Chat/Chat.js

```diff
@@ -10,87 +10,93 @@ import { useSelector } from 'react-redux'
 import { selectActiveConversation } from '../../../state/actions/conversations'
 import FileDialog from './FileDialog'
 
+export const fileTypes = {
+    image: 'image',
+    video: 'video',
+}
+
 const useStyles = makeStyles({
-  root: {
-    flex: 1,
-    display: 'flex',
-    justifyContent: 'space-between',
-    flexDirection: 'column',
-    position: 'relative',
-    width: '100%',
-    height: '100%',
-  },
+    root: {
+        flex: 1,
+        display: 'flex',
+        justifyContent: 'space-between',
+        flexDirection: 'column',
+        position: 'relative',
+        width: '100%',
+        height: '100%',
+    },
 })
 
 const Chat = () => {
-  const classes = useStyles()
-  const listRef = useRef()
-  const [isDragOn, setIsDragOn] = useState(false)
-  const [file, setFile] = React.useState(null)
-
-  const currentUser = useSelector(store => store.auth.user)
-  const activeConversation = useSelector(selectActiveConversation)
-
-  async function onSendMessage(text) {
-    listRef.current.scrollTop = listRef.current.scrollHeight
-    setIsDragOn(false)
-    setFile(null)
-
-    const msgId = uuid()
-
-    const messageRef = db
-      .collection('conversations')
-      .doc(activeConversation.id)
-      .collection('messages')
-      .doc(msgId)
-
-    await messageRef.set({
-      id: msgId,
-      text,
-      ...(file ? { file: 'pending' } : {}),
-      from: currentUser.uid,
-      date: new Date(),
-    })
-
-    if (file) {
-      const fileRef = await storage.ref(`conversations/${activeConversation.id}/${msgId}`).put(file).then((snapshot) => snapshot.ref.getDownloadURL())
-      await messageRef.set({ file: fileRef }, { merge: true })
+    const classes = useStyles()
+    const listRef = useRef()
+    const [isDragOn, setIsDragOn] = useState(false)
+    const [file, setFile] = React.useState(null)
+
+    const currentUser = useSelector(store => store.auth.user)
+    const activeConversation = useSelector(selectActiveConversation)
+
+    async function onSendMessage(text) {
+        listRef.current.scrollTop = listRef.current.scrollHeight
```

Comment:
change attachFile to open drop down with file types options
![image](https://user-images.githubusercontent.com/5127668/67008100-790a1800-f0f1-11e9-8c51-883bd111fae9.png)